### PR TITLE
Storage Protection was renamed to Storage Object in Use Protection

### PR DESCRIPTION
**Pull request by pospispa**
Documentation for PR: https://github.com/kubernetes/kubernetes/pull/59901

Target release: 1.10

### CODE REVIEW

**Comment by pospispa** [comment]
/sig storage
/milestone 1.10
@jsafrane @NickrenREN PTAL

File: docs/admin/admission-controllers.md [hunk 1]
```diff
@@ -414,9 +414,9 @@ This admission controller also protects the access to `metadata.ownerReferences[
...
 subresource of the referenced *owner* can change it.
 
-### Storage Protection (beta)
+### Storage Object in Use Protection (beta)
 {% assign for_k8s_version="v1.10" %}{% include feature-state-beta.md %}
-The `StorageProtection` plugin adds the `kubernetes.io/pvc-protection` or `kubernetes.io/pv-protection` finalizers to newly created Persistent Volume Claims (PVCs) or Persistent Volumes (PV). In case a user deletes a PVC or PV the PVC or PV is not removed until the finalizer is removed from the PVC or PV by PVC or PV Protection Controller. Refer to the [PVC Protection](/docs/concepts/storage/persistent-volumes/#persistent-volume-claim-protection) for more detailed information.
+The `StorageObjectInUseProtection` plugin adds the `kubernetes.io/pvc-protection` or `kubernetes.io/pv-protection` finalizers to newly created Persistent Volume Claims (PVCs) or Persistent Volumes (PV). In case a user deletes a PVC or PV the PVC or PV is not removed until the finalizer is removed from the PVC or PV by PVC or PV Protection Controller. Refer to the [Storage Object in Use Protection](/docs/concepts/storage/persistent-volumes/#storage-object-in-use-protection) for more detailed information.
```
Comment (NickrenREN) [inline docs/admin/admission-controllers.md]: `/docs/concepts/storage/persistent-volumes/#storage-object-in-use-protection` is not a right path

File: docs/concepts/storage/persistent-volumes.md [hunk 2]
```diff
@@ -70,14 +70,14 @@ Pods use claims as volumes. The cluster inspects the claim to find the bound vol
 
 Once a user has a claim and that claim is bound, the bound PV belongs to the user for as long as they need it. Users schedule Pods and access their claimed PVs by including a `persistentVolumeClaim` in their Pod's volumes block. [See below for syntax details](#claims-as-volumes).
 
-### Storage Protection
+### Storage Object in Use Protection
 {% assign for_k8s_version="v1.10" %}{% include feature-state-beta.md %}
-The purpose of the Storage Protection feature is to ensure that Persistent Volume Claims (PVCs) in active use by a pod are not removed from the system as this may result in data loss.
+The purpose of the Storage Object in Use Protection feature is to ensure that Persistent Volume Claims (PVCs) in active use by a pod are not removed from the system as this may result in data loss.
```
Comment (NickrenREN) [inline docs/concepts/storage/persistent-volumes.md]: add PV protection description too ?

File: docs/concepts/storage/persistent-volumes.md [hunk 3]
```diff
@@ -70,14 +70,14 @@ Pods use claims as volumes. The cluster inspects the claim to find the bound vol
...
 
 **Note:** PVC is in active use by a pod when the pod status is `Pending` and the pod is assigned to a node or the pod status is `Running`.
 {: .note}
 
-When the [Storage Protection beta feature](/docs/tasks/administer-cluster/pvc-protection/) is enabled, if a user deletes a PVC in active use by a pod, the PVC is not removed immediately. PVC removal is postponed until the PVC is no longer actively used by any pods.
+When the [Storage Object in Use Protection beta feature](/docs/tasks/administer-cluster/pvc-protection/) is enabled, if a user deletes a PVC in active use by a pod, the PVC is not removed immediately. PVC removal is postponed until the PVC is no longer actively used by any pods.
```
Comment (NickrenREN) [inline docs/concepts/storage/persistent-volumes.md]: ditto

File: docs/admin/admission-controllers.md (hunk 1 above)
Comment (NickrenREN) [reply docs/admin/admission-controllers.md]: we have PVC protection doc as well as PV protection doc, should we merge them into one ?

File: docs/concepts/storage/persistent-volumes.md (hunk 2 above)
Comment (pospispa) [reply docs/concepts/storage/persistent-volumes.md]: @NickrenREN would you mind to add the PV Protection description as you're the author of the source code changes?

File: docs/admin/admission-controllers.md (hunk 1 above)
Comment (pospispa) [reply docs/admin/admission-controllers.md]: Well it behaves strangely for me. Any links to any 3rd level headers didn't work for me, e.g. [this one](https://deploy-preview-7576--kubernetes-io-vnext-staging.netlify.com/docs/concepts/storage/persistent-volumes/#provisioning) or [that one](https://deploy-preview-7576--kubernetes-io-vnext-staging.netlify.com/docs/admin/admission-controllers/#alwayspullimages).
Afterwards, I tried to open it in incognito window and it worked. So I cleared all my browsing files and images and it works also. So you may also have some conflicts with cached files.

File: docs/concepts/storage/persistent-volumes.md (hunk 2 above)
Comment (NickrenREN) [reply docs/concepts/storage/persistent-volumes.md]: of course not .  i will add that

**Comment by NickrenREN** [comment]
@heckj  can we get this merged so that i can update PV protection description based on this ? thanks

**Comment by heckj** [comment]
@NickrenREN merging in now - you should be able to build on this now

**Comment by NickrenREN** [comment]
@heckj  thanks, i have sent a PR for that. see:https://github.com/kubernetes/website/pull/7620

**Pull request merged**